packages/common/src/Constants.ts

```diff
@@ -18,6 +18,15 @@ export const interfaceName = {
   OracleHub: "OracleHub",
 };
 
+// Updated addresses can be found here: https://github.com/makerdao/multicall
+export const MULTICALL2_DEPLOYMENTS: { [chainId: number]: string } = {
+  1: "0x5ba1e12693dc8f9c48aad8770482f4739beed696",
+  42: "0x5ba1e12693dc8f9c48aad8770482f4739beed696",
+  4: "0x5ba1e12693dc8f9c48aad8770482f4739beed696",
+  5: "0x5ba1e12693dc8f9c48aad8770482f4739beed696",
+  3: "0x5ba1e12693dc8f9c48aad8770482f4739beed696",
```

Comment:
few questions: 1) these are all the same. why do we need to store them? 2) should this not be checksum case?

Reply:
but what if the chain doesn't have a deployment for this contract? Should we instead just store a list of networks that do have this contract, and then hardcode the address?

Reply:
Do we not store this in `networks/` because they aren't _our_ system contracts?

If so, should we make this a standard place to store hardcoded external contract addresses by chain id?

contractName -> chainId -> address or chainId -> contractName -> address, maybe?

Reply:
Hm maybe we should add them to `networks` and then we can just call `getAddress()` and if it errors then we can catch the error and log that the multicall contract isn't set for this network.

Reply:
Hmm actually we shouldn't really use `getAddress` within the `common` package because then we'll have a circular dependency between `contracts-node <> common`. I'll go back to hardcoding the multicall address but i'll define it like so:

```
// Updated multicall deployments can be found here: https://github.com/makerdao/multicall
const NETWORKS_WITH_MULTICALL_DEPLOYMENTS = [1, 3, 4, 5, 42];
export const getMulticallAddress = (netId: number): string => {
  if (!NETWORKS_WITH_MULTICALL_DEPLOYMENTS.includes(netId)) throw new Error("No Multicall2 deployment for netId")
  // All Multicall2 addresses are the same.
  return "0x5ba1e12693dc8f9c48aad8770482f4739beed696";
}
```

Reply:
Seems reasonable!

Reply:
Good point on the circular dep.